src/utils/graphQLClient.tsx

```diff
@@ -33,12 +50,53 @@ const sessionId = uuid();
 
 export const client = new Client({
   url: CHROMATIC_API_URL,
-  exchanges: [fetchExchange], // no cacheExchange to prevent sharing data between stories
+  exchanges: [
+    // no cacheExchange to prevent sharing data between stories
```

Comment:
@ghengeveld does this remove the need for this: https://linear.app/chromaui/issue/AP-3675/loading-state-when-switching-story?

Reply:
I don't think so. I didn't change anything in that regard (we didn't have caching before, and we still don't).